src/renderer.js

```diff
@@ -159,25 +159,36 @@ document.addEventListener("DOMContentLoaded", function () {
   var calendar = new FullCalendar.Calendar(calendarEl, {
     initialView: "dayGridMonth",
     dateClick: function (info) {
-      // Use the load dates for the current and day after the current day here
-      monthNumber = info.dateStr[5] + info.dateStr[6];
```

Comment:
I realized that this approach of getting the information for the clicked date is very impractical when it came to working with dates and especially calculating the tomorrow date if the clicked one is at the end of the month so I switched to working with the "Date" data type.